### DISCUSSION TITLE
Who should I contact to ask a permission to publish translated tailwind document on the web?

### DISCUSSION THREAD

**Discussion by AraiEntap** [tailwindlabs/tailwindcss#2922] [General]
This is not the place where I can ask this question but I couldn't find place to ask this.

My company translated TailwindCSS docs into Japanese and would like to publish it on my company website.
I'd like to ask a permission if it's ok or not.
Who and how to ask about it?

Thanks.

**Comment by simonswiss** [comment]
Pinging @adamwathan about this ☝️
